packages/aws-cdk-lib/aws-lambda/test/integ.adot.ts

```diff
@@ -0,0 +1,27 @@
+import * as path from 'path';
```

Comment:
You can remove this file. The test should go in `framework-integ`.

Reply:
Removed